.github/workflows/deploy.yml

```diff
@@ -28,21 +28,12 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GH_TOKEN_CI }}
       - uses: actions/setup-node@v2-beta
         with:
           node-version: '14.17.6'
           registry-url: https://registry.npmjs.org/
 
-      - name: Configure Git
```

Comment:
@JGiter why do we not need this step?

Reply:
I think it is done by actions/checkout. Am I wrong?

Reply:
Build now failed. Is `GH_TOKEN_CI` was provided by devops? I'd like try to update it

Reply:
Hmm, you're right, we're not using this manual git configuration step in ew-credentials https://github.com/energywebfoundation/ew-credentials/blob/develop/.github/workflows/deploy.yml

I added a new `GH_TOKEN` secret. Could you try with this secret instead of `GH_TOKEN_CI`?